src/index.js

```diff
@@ -328,15 +366,15 @@ class JanusAdapter {
   }
 
   getMediaStream(clientId) {
-    var occupantPromise = this.occupantPromises[clientId];
-
-    if (!occupantPromise) {
-      return Promise.reject(
-        new Error(`Subscriber for client: ${clientId} does not exist.`)
-      );
+    if (this.audioStreams[clientId]) {
+      NAF.log.write("Already had audio for " + clientId);
```

Comment:
Should use `debug()` for these. Also I know I wrote the message originally, but I think it confuses people as it sounds kind of like an error, maybe "getMediaStream resolving without waiting for clientId"